lib/Installation/SystemRole/SystemRolePage.pm

```diff
@@ -25,9 +25,14 @@ sub new {
 }
 
 sub init {
-    my $self = shift;
+    my ($self) = @_;
     $self->SUPER::init();
     $self->{sel_role} = $self->{app}->itemselector({id => 'role_selector'});
+    $self->{btn_next} = $self->{app}->button({id => 'next'});
+    $self->{text_mode} = 'Text Mode';
+    $self->{transactional_server} = 'Transactional Server';
```

Comment:
I did similar just using the bare names and I realize that, something like 'role_text_mode', 'role_transactional_server' would be a bit more helpful for the reader as more and more controls are included in the pages, to identify well what are these constants, but up to you, very cosmetic thingy :)